Jumpscale/clients/zero_os_protocol/ZeroOSFactory.py

```diff
@@ -7,7 +7,7 @@ class ZeroOSFactory(j.application.JSBaseConfigsClass):
     """
     """
     _CHILDCLASS = Client
-    __jslocation__ = "j.clients.zos"
+    __jslocation__ = "j.clients.zos_protocol"
```

Comment:
This file also needs to be clean up from method like
- get_by_id
- zero_node_ovh_install
- zero_node_packetnet_install
- get_from_itsyouonline
And everything with is not related to the protocol client.